src/components/tool/guide/Instructions/StartInstructions.js

```diff
@@ -34,7 +35,7 @@ export function LyraStartInstructions({
             In the ssh session, run the following command to schedule the {jobType.toLowerCase()} job:
         </InstructionText>
         <CopyBox>
-            {`qsub${jobType === "Interactive" ? " -I" : ""} -l walltime=1:00:00 -l ${resources.join(":")}`}
+            {`qsub${jobType === "Interactive" ? " -I" : ""} -l walltime=${wallTime?.hour}:${wallTime?.minute}:00 -l ${resources.join(":")}`}
```

Comment:
I think walltime should stay at 1 hour for interactive jobs. When you set the walltime for a batch job and change back to an interactive, the walltime persists rather than reverting to 1 hour.

Reply:
Fixed & Updated